server/routerlicious/packages/lambdas/package.json

```diff
@@ -61,6 +61,7 @@
 		"@fluidframework/server-services-client": "workspace:~",
 		"@fluidframework/server-services-core": "workspace:~",
 		"@fluidframework/server-services-telemetry": "workspace:~",
+		"@fluidframework/server-services-utils": "workspace:~",
```

Comment:
We have specifically not included this `server-services-utils` in the lambdas because that service-utils package brings in `express`, `ioredis`, `winston` and many other packages into the dependency tree. 
Could you move the ApiCounters classes to a different package? The telemetry package sounds appropriate for it

Reply:
Moved to `server-services-telemetry`.